ament_cmake_google_benchmark/package.xml

```diff
@@ -5,6 +5,7 @@
   <version>1.0.1</version>
   <description>The ability to add Google Benchmark tests in the ament buildsystem in CMake.</description>
   <maintainer email="scott@openrobotics.org">Scott K Logan</maintainer>
```

Comment:
@cottsay will you be maintaining this package? I haven't changed this yet, I wasn't sure.

Reply:
I don't think it should be treated any different from the other `ament_cmake` packages, so it should probably be changed to you and @dirk-thomas.

Reply:
Done in https://github.com/ament/ament_cmake/pull/286/commits/800cda31be8b5a6c12b51d332bcb131a4b435af3